docs/apache-airflow-providers-apprise/index.rst

```diff
@@ -0,0 +1,77 @@
+
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+``apache-airflow-providers-apprise``
+======================================
+
+Content
+-------
+
+.. toctree::
+    :maxdepth: 1
+    :caption: Guides
+
+    Connection types <connections>
+
+
+.. toctree::
+    :maxdepth: 1
+    :caption: Commits
+
+    Detailed list of commits <commits>
+
+
+.. toctree::
+    :maxdepth: 1
+    :caption: Resources
+
+    Python API <_api/airflow/providers/apprise/index>
+    PyPI Repository <https://pypi.org/project/apache-airflow-providers-apprise/>
+    Installing from sources <installing-providers-from-sources>
+
+Package apache-airflow-providers-apprise
+-----------------------------------------
+
+`Apprise <https://github.com/caronc/apprise>`__
+
+
+Release: 1.0.0
+
+Provider package
+----------------
+
+This is a provider package for ``apprise`` provider. All classes for this provider package
+are in ``airflow.providers.apprise`` python package.
+
+
+Requirements
+------------
+
+The minimum Apache Airflow version supported by this provider package is ``2.4.0``.
+
+==================  ==================
+PIP package         Version required
+==================  ==================
+``apache-airflow``  ``>=2.2.0``
```

Comment:
```suggestion
``apache-airflow``  ``>=2.4.0``
```

Reply:
@jedcunningham Just curious, why 2.4.0??

Reply:
Short version: 2.4 is the minimum supported Airflow version for providers

https://github.com/apache/airflow/blob/main/PROVIDERS.rst#minimum-supported-version-of-airflow-for-community-managed-providers

Why 2.4 vs 2.6 since this is primarily for the notifier? Well, the hook can be used in 2.4 and 2.5.

Reply:
didn't know the provider's reason. Thanks. :)